csm-extensions/services/dev-mysql/Makefile

```diff
@@ -40,19 +43,20 @@ default: help
 help:
 	@echo "These 'make' targets are available."
 	@echo
-	@echo "  all                  cleans existing container and images and then"
-	@echo "                       build docker image and runs the extension container "
-	@echo "  build                build the extension binary"
-	@echo "  run                  run the csm docker container for extension"
-	@echo "  run-db               run the db behind csm"
+	@echo "  all                  Cleans existing container and images and then"
```

Comment:
Oddly, `all` depends on `build-image` which doesn't appear to be a real target.

Reply:
I am pretty sure that I did not change the `all` target itself. Will check.

Reply:
Ah, my mistake, it's at the bottom of the Makefile.